### ISSUE TITLE
Login in with full name containing "ö" breaks login

### ISSUE THREAD

**Issue by Crown0815** [pydio/cells#441]
How to reproduce in the demo instance:

1. Log in as admin
2. Change "alice"'s full name to, e.g., "Test with ö".
3. Save changes

In my local installation, the error message produced by this process is 

```
pydio-cells-1  | 2022-10-31T14:14:48.603Z       ERROR   pydio.rest.frontend     got a token but failed to load claims   {"error": "rpc error: code = Internal desc = header key \"x-cells-x-pydio-claims\" contains value with non-printable ASCII characters", "SpanUuid": "e5afde1c-449b-420d-8ce2-2e67a2b8d936", "RemoteAddress": "192.168.150.253:53004", "UserAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/106.0.0.0 Safari/537.36", "HttpProtocol": "HTTP/2.0", "UserName": "kroener", "UserUuid": "6ab3916a-9a00-4007-92df-cd8b254faa35", "GroupPath": "/", "Profile": "standard", "Roles": "ROOT_GROUP,ADMINS,6ab3916a-9a00-4007-92df-cd8b254faa35"}
pydio-cells-1  | 2022-10-31T14:14:48.605Z       ERROR   pydio.rest.frontend     Rest Error 500  {"error": "rpc error: code = Internal desc = header key \"x-cells-x-pydio-claims\" contains value with non-printable ASCII characters", "SpanUuid": "e5afde1c-449b-420d-8ce2-2e67a2b8d936", "RemoteAddress": "192.168.150.253:53004", "UserAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/106.0.0.0 Safari/537.36", "HttpProtocol": "HTTP/2.0", "UserName": "kroener", "UserUuid": "6ab3916a-9a00-4007-92df-cd8b254faa35", "GroupPath": "/", "Profile": "standard", "Roles": "ROOT_GROUP,ADMINS,6ab3916a-9a00-4007-92df-cd8b254faa35"}
```

**Comment by cdujeu** [comment]
Nice one ! Thanks for reporting!

**Comment by bsinou** [comment]
This is fixed in the 4.0.3 that has been released yesterday. Thanks again for reporting.
